src/main/java/com/shipmentEvents/handlers/EventHandler.java

```diff
@@ -97,7 +97,7 @@ private void processShipmentUpdates(final LambdaLogger logger) throws Interrupte
         final AmazonS3 s3Client = EventHandler.getS3Client();
         logger.log("Processing Bucket: " + bucketName);
 
-        ObjectListing files = s3Client.listObjects(bucketName); //test-line
+        ObjectListing files = s3Client.listObjects(bucketName); //test line
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses an outdated API. [ListObjectsV2 is the revised List Objects API, and we recommend you use this revised API for new application developments](https://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html).